### PR TITLE
Feat/update sidebar connect button and sidebar toggling

### PR DESCRIPTION
**Pull request by themonster2015**
## Checklist

- [x] I have read the **README** document
- [x] I have read the **CONTRIBUTING** document
- [ ] I have added tests to cover my changes
- [ ] All new and existing tests passed
- [x] My commit message follows [Conventional Commit Guideline](https://conventionalcommits.org/)

### CODE REVIEW

File: ui/widgets/layout/src/components/layout-widget.tsx [hunk 1]
```diff
@@ -154,13 +158,26 @@ const Layout: React.FC<RootComponentProps> = props => {
...
             <div className={tw(sidebarSlotStyle)}>
-              <div className={tw('pt-0 xl:pt-4 h-screen')} ref={sidebarWrapperRef}>
-                <Extension
-                  fullHeight
-                  name={props.layoutConfig.sidebarSlotId}
-                  uiEvents={props.uiEvents}
-                />
-              </div>
+              {needSidebarToggling ? (
+                <div className={tw('pt-0 xl:pt-4 h-screen')} ref={wrapperRef}>
```
Comment (josenriagu) [inline ui/widgets/layout/src/components/layout-widget.tsx]: can you use Box component instead form DS core, then you may pass just plain style strings (without tw) to 'customStyle' prop

File: ui/widgets/layout/src/components/layout-widget.tsx [hunk 2]
```diff
@@ -236,7 +253,7 @@ const Layout: React.FC<RootComponentProps> = props => {
               <Extension name={props.layoutConfig.pluginSlotId} uiEvents={props.uiEvents} />
             </div>
           </div>
-          <div className={tw('sticky top-0 h-screen')}>
+          <div className={tw('sticky top-0')}>
```
Comment (josenriagu) [inline ui/widgets/layout/src/components/layout-widget.tsx]: same here, just replace all divs with Box and and you may safely do away with 'tw'

File: ui/widgets/sidebar/src/components/sidebar-component.tsx [hunk 3]
```diff
@@ -13,6 +13,12 @@ import BasicCardBox from '@akashaorg/design-system-core/lib/components/BasicCard
 import { useGetLogin, useLogout } from '@akashaorg/ui-awf-hooks';
 
 const SidebarComponent: React.FC<RootComponentProps> = props => {
+  const [isMobile, setIsMobile] = React.useState(window.matchMedia('(max-width: 1439px)').matches);
+
+  React.useEffect(() => {
+    setIsMobile(window.matchMedia('(max-width: 1439px)').matches);
+  }, [window.matchMedia('(max-width: 1439px)').matches]);
```
Comment (josenriagu) [inline ui/widgets/sidebar/src/components/sidebar-component.tsx]: can we have these breakpoints (1440, 1439 etc) defined somewhere and imported as variables instead? This makes it easier to update/maintain in future

**Review by josenriagu — CHANGES_REQUESTED** [review]

**Review by quininez — DISMISSED** [review]

**Review by quininez — APPROVED** [review]

File: ui/widgets/sidebar/src/components/sidebar-component.tsx (hunk 3 above)
Comment (themonster2015) [reply ui/widgets/sidebar/src/components/sidebar-component.tsx]: I've defined them somewhere.

**Review by josenriagu — APPROVED** [review]

**Pull request merged**